vault/dendron.tutorial.taking-notes.md

```diff
@@ -2,53 +2,68 @@
 
-This should have created a note named `recipes.md`. Notes in Dendron are just plaintext markdown. They live in your file system and are portable across any platform, which makes it very easy to import/ export your notes and even to publish your notes to a hosted website. You can try to add some content to the body of your `recipes` note now.
+This should have created a note named `recipes.md`. Notes in Dendron are just plaintext [[markdown|dendron://dendron.dendron-site/dendron.topic.markdown]]. They live in your file system and are portable across any platform, which makes it very easy to import/export your notes and even to [[publish your notes to a hosted website|dendron://dendron.dendron-site/dendron.topic.publish]]. You can try to add some content to the body of your `recipes` note now.
 
-The --- section at the top of each note is frontmatter. Frontmatter are custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing. Don't modify the id attribute on the front matter.
+The `---` section at the top of each note is **frontmatter**. [[Frontmatter|dendron://dendron.dendron-site/dendron.topic.frontmatter]] contains custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing.
+
+- Don't modify the `id` or `created` attributes in frontmatter.
+- The `updated` attribute automatically _updates_ whenever you make a new modificaiton to a file.
 
 ### Creating a hierarchy
 
-Dendron uses flexible hierarchies to help you organize your notes. It's how people are able to manage tens of thousands of notes inside Dendron.
+Dendron uses flexible hierarchies to help you organize your notes. [It's how people are able to manage tens of thousands of notes inside Dendron](https://www.kevinslin.com/notes/3dd58f62-fee5-4f93-b9f1-b0f0f59a9b64/).
 
-> 🌱 To create a hierarchy, bring up lookup again `(Ctrl+L / Cmd+L)`. Type `recipes.vegetarian` and then press enter.
+> 🌱 To create a hierarchy, bring up lookup again (`Ctrl+L` / `Cmd+L`). Type `recipes.vegetarian` and then press enter.
 
-You have now created your first hierarchy. Hierarchies in Dendron are created by having a `.` delimiter in the file name. Take a look at the tree view in the side panel to see that the `vegetarian` note exists under the `recipes` hierarchy. You can also see in the Workspace panel that the note file is stored as `recipes.vegetarian.md`.
+You have now created your first **hierarchy**! [[Hierarchies|dendron.topic.hierarchies]] in Dendron are created by having a `.` (a "period" or "dot") delimiter in the file name. Take a look at the Tree View in the side panel to see that the `vegetarian` note exists under the `recipes` hierarchy. You can also see in the Workspace panel that the note file is stored as `recipes.vegetarian.md`.
```

Comment:
link to [[What's with the 's?|dendron.topic.hierarchies#whats-with-the-s]] for explanation of the format

Reply:
replace all occurences of "Workspace panel" with "File Explorer Panel"